backend/apps/volontulo/models.py

```diff
@@ -164,14 +164,18 @@ def create_new(self):
 
     def determine_action_status(self):
         """Determine action status by offer dates."""
+
         if (
                 (
-                    self.finished_at and
-                    self.started_at < timezone.now() < self.finished_at
+                    self.finished_at is None and
```

Comment:
Ok, I see
```
if self.started_at or self.finished_at:
    self.action_status = self.determine_action_status()
```
but I think, that there's no reason to leave this logic outside of `determine_action_status` function - could we move it inside?